sources/mysql/README.md

```diff
@@ -24,41 +24,41 @@ on a MySQL database (via go's database/sql package).
 To use HarbourBridge on a MySQL database called mydb using mysqldump, run:
```

Comment:
This first command just does schema conversion, and then a few examples later there is an example of data conversion in the context of the instance parameter. This is a bit confusing (and isn't the data conversion example incomplete -- don't you need a session file?). I think you need to explicitly introduce schema and data mode. How about something like:

You can directly pipe the output of mysqldump to HarbourBridge. For example:

```sh
mysqldump mydb | harbourbridge schema -source=mysql
```

will build a Spanner schema from the MySQL database 'mydb'. You can also run the tool on an existing mysqldump file:

```sh
harbourbridge schema -source=mysql < my_mysqldump_file
```

Both of these are examples of HarbourBridge schema subcommand. There are two additional subcommands: data and eval. The data subcommand does data conversion, and the eval subcommand performs an all-in-one combinated schema and data conversion.

then give examples of the two commands

and then describe the target-profile options.

Hmm. I guess one issue here is the repetition of this for both postgres and dynamo.

Reply:
I have now added examples of schema, data and oneshot to all the source database specific READMEs. I didn't see the repetition across different READMEs as an issue since each of those READMEs along with the main README should be complete on it's own. It's likely a give user will focus on only one of the source databases at a time. Examples with data subcommand now also talk about requiring session file.